xwiki-platform-distribution/xwiki-platform-distribution-debian/xwiki-platform-distribution-debian-common/pom.xml

```diff
@@ -130,6 +130,17 @@
                 <prefix>/usr/lib/xwiki/META-INF</prefix>
               </mapper>
             </data>
+            <!-- Copyright file -->
+            <data>
```

Comment:
Hmm I think it might work to define this one in the parent and then use `combine.children="append"` in each child `<dataSet>`.